### PR TITLE
MOSIP-32422 Added filter in attributes for pdf card policy

### PR DESCRIPTION
**Pull request by kameshsr**

### CODE REVIEW

**Review by balaji-alluru — APPROVED** [review]

**Review by LoganathanSekar7627 — APPROVED** [review]

**Comment by ckm007** [comment]
Folder name should be db_upgrade_scripts instead of db_release_scripts.

File: db_release_scripts/mosip_pms/sql/1.2.0.1_to_1.2.1.0_release.sql [hunk 1]
```diff
@@ -0,0 +1,15 @@
+-- -------------------------------------------------------------------------------------------------
```
Comment (ckm007) [inline db_release_scripts/mosip_pms/sql/1.2.0.1_to_1.2.1.0_release.sql:1]: We dont need this Description.

Comment (kameshsr) [reply db_release_scripts/mosip_pms/sql/1.2.0.1_to_1.2.1.0_release.sql]: Removed.

File: db_upgrade_scripts/mosip_pms/sql/1.2.0.1_to_1.2.1.0_release.sql
Comment (ckm007) [inline db_upgrade_scripts/mosip_pms/sql/1.2.0.1_to_1.2.1.0_release.sql:1]: file to be renamed to 1.2.0.1_to_1.2.1.0_upgrade.sql

File: db_upgrade_scripts/mosip_pms/sql/1.2.0.1_to_1.2.1.0_revoke.sql
Comment (ckm007) [inline db_upgrade_scripts/mosip_pms/sql/1.2.0.1_to_1.2.1.0_revoke.sql:1]: file to be renamed to 1.2.0.1_to_1.2.1.0_rollback.sql

File: db_upgrade_scripts/mosip_pms/sql/1.2.0.1_to_1.2.1.0_release.sql
Comment (kameshsr) [reply db_upgrade_scripts/mosip_pms/sql/1.2.0.1_to_1.2.1.0_release.sql:1]: Done

File: db_upgrade_scripts/mosip_pms/sql/1.2.0.1_to_1.2.1.0_revoke.sql
Comment (kameshsr) [reply db_upgrade_scripts/mosip_pms/sql/1.2.0.1_to_1.2.1.0_revoke.sql:1]: Done

**Review by ckm007 — APPROVED** [review]

**Pull request merged**